project/Dependencies.scala

```diff
@@ -19,10 +19,12 @@ object Dependencies {
   val junitVersion = "4.13.2"
   val slf4jVersion = "1.7.36"
   // check agrona version when updating this
+  // Note: 1.46 is JDK 17 only so we cannot bump until we stop supporting JDK 11
   val aeronVersion = "1.44.1"
   // needs to be inline with the aeron version, check
   // https://github.com/real-logic/aeron/blob/1.x.y/build.gradle
-  val agronaVersion = "1.19.2"
+  // Note: 1.23+ is JDK 17 only so we cannot bump until we stop supporting JDK 11
+  val agronaVersion = "1.22.0"
```

Comment:
looking at `Test/dependencyTree` in akka-cluster
```
[info]   | +-io.aeron:aeron-client:1.44.1
[info]   | | +-io.aeron:aeron-annotations:1.44.1
[info]   | | +-org.agrona:agrona:1.23.0
[info]   | | +-org.agrona:agrona:[1.21.1,2.0) (evicted by: 1.23.0)
```

I wonder where that agrona 1.23.0 comes from?
Also, why did this start failing 5 days ago? https://github.com/akka/akka/actions/workflows/nightly-builds.yml

Reply:
It seems the range `[1.21.1,2.0)` means that we already were on 1.22 but we never noticed once they released JDK 17+ 1.23 it bumped for us. Some sbt resolution rule where it wins because range incompatible with our direct dependency I had not seen before.

Reply:
Incompatible as: we depended on 1.19 but range on aeron said it needed 1.21 minimum.

Reply:
wow, scary. dependencyTree is looking good in this branch

Reply:
1.23.0 was released 5 days ago.

It looks like when there's an explicit dependency on version x, but a transitive dependency on > y where y > x, sbt chooses the > y dependency.  So we were pulling in 1.22.0 for some time before this and 1.21.x before that despite our explicit dependency on 1.19.2.